app/state.go

```diff
@@ -1843,3 +1843,230 @@ func setBrokenArtifactFlag(store store.Store, artName string) {
 		// already in an error path.
 	}
 }
+
+///////////////////////
+// Control map state //
+///////////////////////
+
+type Wrap struct {
+	state UpdateState
+}
+
+type controlMapState struct {
+	wrappedState UpdateState
+	baseState
+}
+
+func NewControlMapState(wrapsState Wrap) State {
```

Comment:
Isn't `NewControlMapState(wrapsState State)` essentially the same thing?

Reply:
Yes, it's exactly the same, I was just experimenting with the interface a little

Reply:
Changing it back to the default (sane option).
Just wanted to see how it read :)